### DISCUSSION TITLE
Some Twitter blogs can't be downloaded

### DISCUSSION THREAD

**Discussion by Klimax** [TumblThreeApp/TumblThree#483] [General]
**Describe the bug**
Blog https://twitter.com/CalibreObscura can't be downloaded. While it exists, app reports "the blog is offline" and HTTP result for IsBlogOnline is 403.

**To Reproduce**
1. Add https://twitter.com/CalibreObscura to blogs to download
2. Try to downloaded

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - TumblThree version: v2.11.3
 - OS: Windows 10 pro, fully updated

**Additional context**
It is possible I am doing something wrong, but it is not possible to figure it out based on available documentation.

**Comment by thomas694** [comment]
Twitter download works, this blog too.
If you get a "403 Forbidden" it normally means that you are not logged in. Please go to the settings dialog and log in.

**Comment by Klimax** [comment]
I'm sorry. I mistook it for old newTumblr login that used to occupy that spot. Nothing to see here, move along... ;)

Just a thought, it might be good idea to mention it in User Guide.
